test/infrastructure/docker/go.mod

```diff
@@ -8,13 +8,13 @@ require (
 	github.com/pkg/errors v0.9.1
 	github.com/spf13/pflag v1.0.5
 	k8s.io/api v0.17.9
-	k8s.io/apimachinery v0.17.9
+	k8s.io/apimachinery v0.18.8
```

Comment:
We shouldn't update these deps

Reply:
second this.
Let's keep this backport scoped down to changes in #3815 and #4002